src/models/Campaign.js

```diff
@@ -50,7 +50,7 @@ class Campaign extends BasicModel {
     this.archivedMilestones = new Set(data.archivedMilestones || []);
     this.customThanksMessage = data.customThanksMessage;
     this.slug = data.slug;
-    this._gasPaidUsdValue = data.gasPaidUsdValue || '0';
+    this._gasPaidUsdValue = data.gasPaidUsdValue || 0;
```

Comment:
value 0 is resolved as false in condition like this
```
            {currentUser.gasPaidUsdValue && (
```
I intentionally made this and other default value to string type